spyder/plugins/editor/lsp/transport/common/consumer.py

```diff
@@ -124,6 +124,8 @@ def run(self):
     def parse_headers(self, headers):
         logger.debug(headers)
         headers = headers.split(b'\r\n')
+        headers = [header.split(b'\n')[-1] if b'\n' in header else header
+                   for header in headers]
```

Comment:
Why this change? And also, is the line above this one (i.e. `headers = headers.split(b'\r\n')`) necessary? Or can it be removed?